engine/gamesys/src/gamesys/resources/res_font.cpp

```diff
@@ -344,6 +382,12 @@ namespace dmGameSystem
 
     dmResource::Result ResFontAddGlyph(FontResource* font, uint32_t codepoint, FontGlyph* inglyph, void* imagedata, uint32_t imagedatasize)
     {
+        DynamicGlyph** glyphp = font->m_DynamicGlyphs.Get(codepoint);
+        if (glyphp != 0)
+        {
+            return dmResource::RESULT_OK;
+        }
```

Comment:
It's already added, don't add it again.
It takes time, and it's already added?!